keyboards/keebio/bdn9/keymaps/marianas/keymap.c

```diff
@@ -0,0 +1,92 @@
+/* Copyright 2019 Danny Nguyen <danny@keeb.io>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum layers {
+  BASE,
+  MEDIA
+};
+
+enum keys {
+  MODESW = SAFE_RANGE,
+};
+
+uint32_t layer = 0;
+
+bool updateLayerState(uint16_t keycode, keyrecord_t* record) {
+  if (record-> event.pressed)
+  {
+    switch (keycode)
+    {
+      case MODESW:
```

Comment:
You could remove this completely and use the `TG` keycode.

Or use `layer_invert`.